<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arduloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="5">
     <widget class="QLabel" name="headLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="5">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="hexfileCombox">
        <property name="minimumSize">
         <size>
          <width>311</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openHexButton">
        <property name="text">
         <string>Browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="portCombox">
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="uploadButton">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QComboBox" name="mcuCombox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
